test/infrastructure/vm-operator/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Maybe this should be at `hack/vm-operator` ?

Reply:
I'm fine to keep it as it is for the first iteration